cloudferrylib/os/actions/check_cloud.py

```diff
@@ -163,9 +165,17 @@ def run(self, **kwargs):
             LOG.error(err_message)
             raise exception.AbortMigrationError(err_message)
         compute_resource.nova_client.servers.delete(vm_new_ids.keys()[0])
+        for i in xrange(LIMIT_RETRY_CHECK_INSTANCE):
+            if compute_resource.nova_client.servers.list(vm_new_ids.keys()[0]):
+                time.sleep(1)
```

Comment:
Where it is specified?
Here is the description of this option from `configs/config.ini`:
> # Time in seconds between Openstack API retries.
> # Note: High number may considerably slow down migration process.
https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/configs/config.ini#L111-L112

And also from `cfglib.py`:
> Time wait if except Performing error
https://github.com/MirantisWorkloadMobility/CloudFerry/blob/devel/cfglib.py#L139

There is nothnig related to proxy-client there.